eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/ArrayListAdapter.java

```diff
@@ -238,13 +238,6 @@ public void sort(Comparator<? super T> comparator)
         return (ArrayListAdapter<T>) super.sortThis();
     }
 
-    @Override
-    public ArrayListAdapter<T> with(T element)
```

Comment:
Unfortunately, this is a breaking change. It's a good idea, it just has to wait for the next major version.

Reply:
This needs to be removed for minor version